../pipewire-0.3.80/src/modules/module-protocol-pulse/pulse-server.c:211:64: warning: format specifies type 'unsigned long' but the argument has type 'uint64_t' (aka 'unsigned long long') [-Wformat]

Index: src/modules/module-protocol-pulse/pulse-server.c
--- src/modules/module-protocol-pulse/pulse-server.c.orig
+++ src/modules/module-protocol-pulse/pulse-server.c
@@ -208,7 +208,7 @@ static int send_object_event(struct client *client, st
 {
 	uint32_t event = 0, mask = 0, res_index = o->index;
 
-	pw_log_debug("index:%d id:%d %08lx type:%u", o->index, o->id, o->change_mask, type);
+	pw_log_debug("index:%d id:%d %08llx type:%u", o->index, o->id, o->change_mask, type);
 
 	if (pw_manager_object_is_sink(o) && o->change_mask & PW_MANAGER_OBJECT_FLAG_SINK) {
 		client_queue_subscribe_event(client,
